src/network/networkd-util.c

```diff
@@ -99,3 +99,48 @@ int config_parse_address_family_boolean_with_kernel(
 
         return 0;
 }
+
+int ip_port_from_string(const char *s, uint16_t *port) {
```

Comment:
We have `parse_ip_port` already, so this function is unnecessary.

Reply:
@zx2c4 this will return invalid argument on `port = 0`. I suppose this is ok.